run-make-check.sh

```diff
@@ -80,7 +80,7 @@ function main() {
         fi
     done
     # Init defaults after deps are installed.
-    local cmake_opts=" -DCMAKE_CXX_COMPILER=$cxx_compiler -DCMAKE_C_COMPILER=$c_compiler -DWITH_GTEST_PARALLEL=ON -DWITH_FIO=ON -DWITH_CEPHFS_SHELL=ON -DWITH_SPDK=ON -DENABLE_GIT_VERSION=OFF"
+    local cmake_opts=" -DCMAKE_CXX_COMPILER=$cxx_compiler -DCMAKE_C_COMPILER=$c_compiler -DWITH_GTEST_PARALLEL=ON -DWITH_CEPHFS_SHELL=ON -DWITH_SPDK=ON -DENABLE_GIT_VERSION=OFF"
```

Comment:
this change practically reverts part of #20535 .


see https://github.com/ceph/ceph/pull/20535/files#diff-54793ec7325559f136d18596b00188c00c48dbb1c253a20cb327b80092201edeR71

i'd suggest just bump its version to the latest release.